pkg/virt-controller/watch/migration.go

```diff
@@ -1107,6 +1151,13 @@ func (c *MigrationController) sync(key string, migration *virtv1.VirtualMachineI
 		return fmt.Errorf("vmi is inelgible for migration because another migration job is running")
 	}
 
+	if err = c.handleMigrationBackoff(key, vmi, migration, pod); err != nil {
+		warningMsg := fmt.Sprintf("backoff migrating vmi %s/%s", vmi.Namespace, vmi.Name)
+		c.recorder.Eventf(vmi, k8sv1.EventTypeWarning, err.Error(), warningMsg)
+		log.Log.Object(migration).Warning(warningMsg)
+		return err
```

Comment:
Do you want to return error here?

Reply:
yes, that's because I don't want to continue if the VM is currently in MigrationBackoff

Reply:
But you could just return nil, or not? This will cause you to enqueue it again. Generally, you might want to reuse the already present enqueue mechanism but this needs to be explored.

Reply:
It won't be re-enqueued because of https://github.com/kubevirt/kubevirt/pull/8530/files#diff-1baf192bb9c4560d7d40f008b7ba5c680ebce669b8f1b7639445abd3e7269688R218.

Reply:
nit: I think you could just return nil here and remove the line mentioned above

Reply:
Yes, you're right, I can actually simplify that. Thank you

Reply:
Done!